runelite-client/src/main/java/net/runelite/client/plugins/inventorygrid/InventoryGridConfig.java

```diff
@@ -60,4 +60,14 @@ default boolean showHighlight()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "overlayDelay",
```

Comment:
should this use the old config key?

Reply:
I don't have a strong opinion about it, i just figured that the behavior for this config was sufficiently different than that of the old one. If you want to we can just use the old one.

Reply:
It looks to me like the behavior isn't different though?

Reply:
I guess it depands on how you see it, for me the old delay was used by the plugin to determine if you were dragging an item. Now, the dragging check is done by checking the amount of pixels dragged. And the new delay is used to determine how much time has to pass before showing any overlay. But i'll have to rebase anyway, so i'll just change it back.